README.md

```diff
@@ -28,6 +28,7 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 | Name           | Default Value | Description                        |
 | -------------- | ------------- | -----------------------------------|
 | `prometheus_version` | 2.16.0 | Prometheus package version. Also accepts `latest` as parameter. Only prometheus 2.x is supported |
+| `prometheus_install_method` | "auto" | Prometheus installation method takes `auto` or `manual` where `manual` effectively skips installation. |
```

Comment:
Let's switch this to boolen like `prometheus_skip_install` and set it to `false` by default.

Reply:
@paulfantom, contact me if @DEvil0000 does not respond to your request

Reply:
I feel like I did ;)

Reply:
It's mean that you will not update this MR anymore ?

Reply:
excuse me if I was not clear. I updated it and requested a new review before your comment as you can see in its history.
So what do you think is missing?

Reply:
OK. Thanks.  @paulfantom, is-it ok for you ?